examples/auth/envoy_config.json

```diff
@@ -442,6 +442,20 @@
                                 "requirementName": "1.examples_auth_endpoints_cloudesf_testing_cloud_goog.CreateShelf"
                               }
                             }
+                          },
+                          {
+                            "decorator": {
+                              "operation": "ingress UnknownMethod"
+                            },
+                            "directResponse": {
+                              "body": {
+                                "inlineString": "The request is not defined by this API."
+                              },
+                              "status": 404
```

Comment:
Why do we need to add such catch all?   It is done automatically by envoy router [decodeHeader](https://github.com/envoyproxy/envoy/blob/master/source/common/router/router.cc#L361)

Reply:
Didnt release that. Thanks!